app/schemas/signedUpUserSchema.js

```diff
@@ -0,0 +1,48 @@
+const {
+  EMAIL_DOMAIN,
+  PASSWORD_FORMATS,
+  MIN_LENGTH,
+  invalidEmailDomainMessage,
+  invalidPasswordMessage,
+  invalidPasswordLengthMessage
+} = require('../helpers');
+
+exports.signedUpUserSchema = {
+  email: {
+    in: ['body'],
+    isEmpty: {
+      errorMessage: 'missing_field',
+      negated: true
+    },
+    isEmail: {
+      errorMessage: 'Invalid email'
+    },
+    matches: {
+      options: [EMAIL_DOMAIN],
+      errorMessage: invalidEmailDomainMessage
+    }
+  },
+  password: {
+    in: ['body'],
+    isEmpty: {
+      errorMessage: 'missing_field',
+      negated: true
+    },
+    custom: {
+      options: value =>
+        PASSWORD_FORMATS.reduce((result, format) => {
+          if (!value.match(format)) {
+            return false;
+          }
+          return result;
+        }, true),
```

Comment:
I see this password validator before, can we abstract this too ? Like a common in schemas and share between differents schemas, maybe signup and signin schema could be in same file (like an users schema)